scripts/monitoring-mainnet.conf

```diff
@@ -0,0 +1,24 @@
+[default]
+agent-address=
+deployment-dir="../deployments/mainnet"
+
+[notification.rocketchat]
+enabled=false
+url=
+channel=
+
+[notification.telegram]
+enabled=true
+url=
+
+[chains.arbitrum]
+rpc-url=
+explorer=https://arbiscan.io/tx/
+chain-id=42161
+start-block=46158611
```

Comment:
maybe better to get this info in script using deployment.json ? so we dont need to change conf with every possible release

Reply:
same true for chain_id as you can get it with w3 object like `w3.eth.chain_id` but as it is not changing not so important

Reply:
It would be kind of hard to get the start_block from the deployments.json. 

First I'll have to get the chain_id for the given rpc, then look inside of the deployments for that chainid, then for the RequestManager and then block_number. 

Of course it is doable, but sounds like useless code if those values are hardly ever going to change.

Reply:
Well we are doing it already in agent.py but if others say ok then it is ok.